cli/cmd/inject.go

```diff
@@ -304,6 +310,12 @@ func (resourceTransformerInject) generateReport(reports []inject.Report, output
 		output.Write([]byte(fmt.Sprintf("%s %s\n", okStatus, udpDesc)))
 	}
 
+	if len(automountServiceAccountTokenFalse) > 0 {
+		output.Write([]byte(fmt.Sprintf("%s \"automountServiceAccountToken: false\" detected in %s\n", warnStatus, strings.Join(automountServiceAccountTokenFalse, ", "))))
```

Comment:
For my own understanding, `output` writes to `stderr`, right? But I can't seem to get the CLI to output this error. What am I missing?

Reply:
Ah, I think this might be redundant. The CLI will error out before this part of the code is executed.

Reply:
Let's remove it then. Thanks.

Reply:
Done :+1: